.molecule/default/create_docker.yml

```diff
@@ -0,0 +1,120 @@
+# this is mainly a copy of https://github.com/ansible-community/molecule-docker/blob/master/molecule_docker/playbooks/create.yml
+# with fix: https://github.com/ansible-community/molecule-docker/pull/30
```

Comment:
I notice that this was closed https://github.com/ansible-community/molecule-docker/pull/30

the pr included both a bug fix and an addition - was anything done in this direction so that we can go back to the default molecule playbooks?
do we need to have this yml in the aiida-core repo?

Reply:
Well the key thing is that we need to be able to set the Docker context (see `context: "../.."` above in `.molecule/default/config_local.yml`), i.e. setting `docker.image.path`.
I've re-opened in https://github.com/ansible-community/molecule-docker/pull/37, without the bug fixes (which maybe were not correct) to hopefully make it more easy to be accepted.
I imagine it will take a while to make its way into a release though, so might have to open an issue to remove at a later date?

Reply:
sounds good - it would be great if we can avoid maintaining this file (perhaps open an issue on aiida-core for this)

Reply:
opened #4718